src/nbl/asset/utils/IShaderCompiler.cpp

```diff
@@ -119,7 +189,7 @@ auto IShaderCompiler::CIncludeFinder::getIncludeStandard(const system::path& req
 
     core::blake3_hasher hasher;
     hasher.update((uint8_t*)(retVal.contents.data()), retVal.contents.size() * (sizeof(char) / sizeof(uint8_t)));
-    retVal.hash = { *static_cast<core::blake3_hash_t>(hasher).data };
+    retVal.hash = static_cast<core::blake3_hash_t>(hasher);
```

Comment:
instead of putting the code that does header hashing in two places that handle finding includes (which also don't know if you're running with a read or write preprocessing cache at all), is it possible to hash outside/later  in `CCache::find_impl` ?